deploy.sh

```diff
@@ -13,16 +13,15 @@ set -e
 # .env.qa
 # .env.development
 # .env.production
-if [ -f .env.$4 ]
+if [ -f .env.$3 ]
 then
-    export $(cat .env.$4 | sed 's/#.*//g' | xargs)
+    export $(cat .env.$3 | sed 's/#.*//g' | xargs)
 fi
 
 print_usage_and_exit () {
-    echo 'Usage: ./deploy.sh --clear-data=yes|no --restore-metadata=yes|no --update-metadata=yes|no HOST ENV VERSION COUNTRY_CONFIG_VERSION COUNTRY_CONFIG_PATH REPLICAS'
+    echo 'Usage: ./deploy.sh --clear-data=yes|no --restore-metadata=yes|no HOST ENV VERSION COUNTRY_CONFIG_VERSION COUNTRY_CONFIG_PATH REPLICAS'
```

Comment:
Just a thought I had, no action needed: I wonder if this should be considered as a breaking change? We are using this deploy script in Nigeria a certain way (In CI pipelines) and updating to this version would force us to update our code

Reply:
Yeah we are kinda changing the deploy api, so it should be considered a breaking change I guess